corehq/apps/export/views.py

```diff
@@ -2415,3 +2418,23 @@ def download_daily_saved_export(req, domain, export_instance_id):
     return build_download_saved_export_response(
         payload, export_instance.export_format, export_instance.filename
     )
+
+
+class CopyExportView(View):
+    urlname = 'copy_export'
+
+    @method_decorator(login_and_domain_required)
+    def dispatch(self, request, *args, **kwargs):
+        if not self.request.couch_user.can_edit_data():
+            raise Http404
+        else:
+            return super(CopyExportView, self).dispatch(request, *args, **kwargs)
+
+    def get(self, request, domain, export_id, *args, **kwargs):
+        try:
+            export = get_properly_wrapped_export_instance(export_id)
+            export.copy_export()
```

Comment:
nit this could go in the `else` block of the try statement since it won't throw a ResourceNotFound error